.azure-pipelines/generation-pipeline.yml

```diff
@@ -498,6 +498,60 @@ stages:
              migration: false
              pathToCopy: '/com/Beta/*'
 
+- stage: stage_php_beta_kiota
+  dependsOn:
+    - stage_build_and_publish_kiota
+    - stage_beta_openapi
+  condition: |
+    and
+    (
+      eq(dependencies.stage_build_and_publish_kiota.result, 'Succeeded'),
+      eq(dependencies.stage_beta_openapi.result, 'Succeeded')
+    )
+  jobs:
+    - job: php_beta_kiota
```

Comment:
Any chance you can post a link to successful run of the pipeline when generating the beta stage as well to avoid any possible regression?

Reply:
Done

Reply:
Thanks!